CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.6.0 - Refactorings
```

Comment:
Because one can't be sure of the release number beforehand, the convention is to title the section "unreleased":
```suggestion
## Unreleased
```